src/scripts/modules/media/footer/footer.less

```diff
@@ -21,6 +21,7 @@
       display: inline-block;
       padding: 0.7rem 1rem;
       margin-right: 0.3rem;
+      margin-top: 3px;
```

Comment:
Can we use `rem` here, since margin-right and paddings are in rems too?

Reply:
I agree, that would be a good idea to keep the unit consistent